docs/changelog.rst

```diff
@@ -224,7 +224,8 @@ Other new things:
   Thanks to :user:`arogl`.  :bug:`3906`
 * Get ISRC identifiers from musicbrainz
   Thanks to :user:`aereaux`.
-* :doc:`/plugins/metasync`: The ``metasync`` plugin now also fetches the ``Date Added`` field from iTunes databases and stores it in the``itunes_dateadded`` field.Thanks to :user:`sandersantema`.
+* :doc:`/plugins/metasync`: The ``metasync`` plugin now also fetches the ``Date Added`` field from iTunes databases and stores it in the``itunes_dateadded`` field.Thanks to :user:`sandersantema`.* The ``mbsync`` plugin now also fetches performers, if the ``performer_info``
+  option is enabled. Fixes :bug:`1547`, thanks to :user:`dosoe`.
```

Comment:
I'm not sure this does actually fix #1547… that one seems like more of a blanket issue for fetching all kinds of other metadata, and it's not really about mbsync?

Reply:
This issue isn't about mbsync, but this PR fixes it as it fetches most of the extra fields from MusicBrainz (with the exception of performance dates and places).